hadongun/수학/treasure.cpp

```diff
@@ -0,0 +1,48 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <string.h>
+#include <iostream>
+#include <algorithm>
+#include <vector>
+int main(void)
+{
+	int N = 0;
+	scanf("%d\n", &N);
+	int A[50] = { 0 };
+	int B[50] = { 0 };
+	
+	
+	for (int i = 0; i < N; i++)
+	{
+		scanf("%d", &A[i]);
+	}
+	
+	for (int i = 0; i < N; i++)
+	{
+		scanf("%d", &B[i]);
+	}
+	
+	int n = sizeof(A) / sizeof(A[0]);
+	std::sort(A, A + N);
+
+	int C[50] = { 0 };
+	for (int i = 0; i < N; i++)
+	{
+		C[i] = B[i];
+	}
+	int k = sizeof(B) / sizeof(B[0]);
+	std::sort(B, B + N, std::greater<int>());
```

Comment:
배열 A, B를 vector로 선언하면 `rbegin()`, `rend()`를 사용하여 다음과 같이 작성하여 내림차순으로 정렬할 수 있습니다!
[rbegin(), rend()](https://cppmagister.tistory.com/32)
```suggestion
	std::sort(B.rbegin(), B.rend());
```

Reply:
옷!! 역시 대현석님이네요